core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -657,7 +657,8 @@ public static String getInferredAlterationsKnownEffect(String inferredAlt) {
             return new ArrayList<>();
         }
 
-        if (alteration.getConsequence() != null && alteration.getConsequence().equals(VariantConsequenceUtils.findVariantConsequenceByTerm("missense_variant"))) {
+        if (alteration.getConsequence() != null && alteration.getConsequence().equals(VariantConsequenceUtils.findVariantConsequenceByTerm("missense_variant"))
+            && !alteration.getProteinStart().equals(-1) && !alteration.getProteinStart().equals(100000)) {
```

Comment:
What's the meaning of -1 and 100000 in here?

Reply:
It's the default of protein start/end if it does not have any information. This probably should be set to a constant. Let me update the PR.